File: problemas/iniciante/2787/2787.py
def main():
    L = int(input())
    C = int(input())

    tabuleiro = []
    for linha in range(1, L + 1):
        for coluna in range(1, C + 1):
            if (linha - coluna) % 2 == 0:
                tabuleiro.append(1)
            else:
                tabuleiro.append(0)

    print(tabuleiro[-1])

if __name__ == "__main__":
    main()

File: problemas/iniciante/2787/2787.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let lines = input.split('\n');

    let L = parseInt(lines.shift()!.trim());
    let C = parseInt(lines.shift()!.trim());

    let tabuleiro: number[] = [];

    for (let i = 1; i <= L; ++i) {
        for (let j = 1; j <= C; ++j) {
            if ((i - j) % 2 === 0) {
                tabuleiro.push(1);
            } else {
                tabuleiro.push(0);
            }
        }
    }

    console.log(tabuleiro[tabuleiro.length - 1]);
});
